cg/meta/transfer/lims.py

```diff
@@ -84,6 +94,18 @@ def transfer_samples(
             else:
                 LOG.debug(f"no {status_type.value} date found for {sample_obj.internal_id}")
 
+    def _remove_older_orders(
+        self, samples: list[Sample], order_date_cutoff: datetime
+    ) -> list[Sample] | None:
+        no_samples_before_filtering: int = len(samples)
+        for sample_obj in samples:
+            if self._is_sample_ordered_before_order_date_cutoff(
+                sample_obj=sample_obj, order_date_cutoff=order_date_cutoff
+            ):
+                samples.remove(sample_obj)
+        LOG.info(f"Filter out {no_samples_before_filtering-len(samples)} samples")
```

Comment:
```suggestion
        LOG.debug(f"Filter out {no_samples_before_filtering-len(samples)} samples")
```
I do not think this is something we might want to know on a day-to-day basis, rather than as validation once it goes live and if something seems off.